index.test.js

```diff
@@ -79,8 +79,8 @@ describe("AmazonAdsFormatter", () => {
 
   describe("formatPostal", () => {
     it("should format postal codes correctly", () => {
-      expect(formatter.formatPostal("12345-6789")).toBe("12345");
-      expect(formatter.formatPostal("K1A 0B1")).toBe("k1a0b");
```

Comment:
have a test case with postal code of 3 charcters

Reply:
Ok and it should return empty string.
Empty string for postal is not ideal but 3 char postal code is also not ideal either customer has already formatted from their end or postal is wrong. 
Since we ask customer to not format it, empty string should work

Reply:
done